app/views/users/backup_code_setup/confirm_backup_codes.erb

```diff
@@ -0,0 +1,30 @@
+<% title t('titles.backup_codes') %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.backup_codes')) %>
+
+<p class='margin-top-1 margin-bottom-4'>
+  <%= t('two_factor_authentication.backup_codes.warning_html') %>
+</p>
+
+<p class='margin-top-1 margin-bottom-7'>
+  <%= t('two_factor_authentication.backup_codes.instructions', app_name: APP_NAME) %>
+</p>
+
+<div class="border-bottom border-primary-light margin-bottom-2">
```

Comment:
Could we wrap the link with `PageFooterComponent` instead of applying the margins and borders here?